README.md

```diff
@@ -140,6 +140,9 @@ There's nothing to say about Alcatel really. They have fastboot removed from mos
 ### Apple
 This one is probably expected. No iPhone, iPad, iPod Touch or Apple TV model has had an unlockable bootloader. Depending on your device or SoC, there are BootROM (hardware based) and iBoot (software based) exploits you can use to run unsigned code on iOS devices, such as [checkm8 for A5-A11,](https://theapplewiki.com/wiki/Checkm8_Exploit) [limera1n for A4, iPod touch 3G, and iPhone 3GS](https://theapplewiki.com/wiki/Limera1n_Exploit) [24kpwn for certain iPod touch 2G's and iPhone 3GS's](https://theapplewiki.com/wiki/0x24000_Segment_Overflow) and [steaks4uce for iPod touch 2G's](https://theapplewiki.com/wiki/Usb_control_msg(0xA1,_1)_Exploit) for BootROM exploits, and [DRA for all 32-bit devices running iOS 7.x.x](https://theapplewiki.com/wiki/De_Rebus_Antiquis) for iBoot exploits.
 
+### Energizer
+Not only will Energizer destroy your electronics by leaking battery acid everywhere, but they will also destroy your privacy by not allowing you to unlock your bootloader. There's not much more to say, neither their Android phones or KaiOS phones are unlockable.
```

Comment:
> leaking battery acid everywhere

...Did this actually happen?

Reply:
was meant to just be a joke about the unreliability of their AA/AAA batteries, i can remove it

Reply:
It's fine, but it'd just have more weight if it did.